src/ssh-compute.ts

```diff
@@ -0,0 +1,183 @@
+/*
```

Comment:
I know it's what we do elsewhere, but I think we should move this into `main.ts`. There's not a ton of value having it as a separate file since we've figured out good ways to stub and test things.